starter_templates/haskell/code/.codecrafters/run.sh

```
@@ -8,4 +8,4 @@
 
 set -e # Exit on failure
 
-exec $(stack path --local-install-root)/bin/hs-grep-clone-exe "$@"
\ No newline at end of file
+exec $(stack path --local-install-root)/bin/codecrafters-grep-exe "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)